nav2_motion_primitives/CMakeLists.txt

```diff
@@ -73,7 +77,7 @@ if(BUILD_TESTING)
   find_package(ament_cmake_gtest REQUIRED)
   find_package(ament_lint_auto REQUIRED)
   ament_lint_auto_find_test_dependencies()
-  add_subdirectory(test)
+  # add_subdirectory(test)
```

Comment:
Why commented out?

Reply:
Haven't updated the tests, will do on a separate PR.